cmd/utils.go

```diff
@@ -31,19 +31,19 @@ func scanForProjectFiles(ideType IDEType) ([]string, error) {
 
 	var paths []string
 	{
-		if ideType == xcodeIDE {
+		if projectType == iOSProjectType {
```

Comment:
Consider using polymorphism instead of type based branching.

Reply:
I agree with the idea.
But I think we should do this in a separated PR because it would be a bigger refactor.
I've created a card for it.